### PR TITLE
Update to structure

### PR DESCRIPTION
**Pull request by muydanny**
- created web scraping function in Jupyter using Regular Expression

- Using the sample Flask API we built on top of it catering to what we needed (BallotDetails)

-  Successful API call 

Co-authored-by: Rostam <rostammahabadi@gmail.com>, Mariana <cidmariana5@gmail.com>

### CODE REVIEW

**Review by cwefso — APPROVED** [review]
I barely follow what's happening here but I trust you crazy fools

**Comment by muydanny** [comment]
A lot of new technologies being used. We can barely follow too lol But essentially our python API is scraping the site and its sick.

!(https://media.giphy.com/media/Lq17J2gDfmI8rP24uM/giphy.gif)

File: co_ga_scraper.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+from urllib.request import Request, urlopen
+from bs4 import BeautifulSoup
+import re
+
```
Comment (Mariana-21) [inline co_ga_scraper.py:4]: Nice glad you guys got the due dates and stuff!

File: sos_scraper.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
+from urllib.request import Request, urlopen
+from bs4 import BeautifulSoup
+import re
+
```
Comment (Mariana-21) [inline sos_scraper.py:4]: Looks great!

**Review by Mariana-21 — APPROVED** [review]

**Pull request merged**